wrappers/javascript/aries-askar-react-native/cpp/ariesAskar.cpp

```diff
@@ -1016,7 +1016,7 @@ jsi::Value keyEntryListGetMetadata(jsi::Runtime &rt, jsi::Object options) {
       askar_key_entry_list_get_metadata(keyEntryListHandle, index, &out);
   handleError(rt, code);
 
-  return jsi::String::createFromAscii(rt, out);
+  return jsi::String::createFromAscii(rt, out ? out : "{}");
```

Comment:
```suggestion
  if out == nullptr {
    return jsi::Value::undefined();
  } else {
    return jsi::String::createFromAscii(rt, out);
  }
```

Reply:
I changed the return type as suggested, and did the same with `entryListGetValue` and `entryListGetTags`. Seems to be working fine with the RN sample app from [test repo](https://github.com/TimoGlastra/shared-components-react-native-example).